### PR TITLE
Fix malwarebytes-anti-malware.rb

### PR DESCRIPTION
**Pull request by Eitot**
• Fix broken download URL
• Add former name
• Add minimum OS
• Add zap stanza

### CODE REVIEW

**Comment by Eitot** [comment]
Accidentally removed the other pull request. Sorry about that!

File: Casks/malwarebytes-anti-malware.rb [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+  name 'AdwareMedic' # former name
+  appcast 'https://data-cdn.mbamupdates.com/v1/mbam-mac/updates.xml',
           :sha256 => '065d31d146031074b4e249a8994fffedf589fffbc114ad3c6bb98c2195878de4',
           :format => :sparkle
   homepage 'https://www.malwarebytes.org/antimalware/mac/'
   license :gratis
+  depends_on :macos => '>= :lion'
 
   app 'Malwarebytes Anti-Malware.app'
+  zap :delete => [
```
Comment (vitorgalvao) [inline Casks/malwarebytes-anti-malware.rb]: Alignment should be:

```ruby
  zap :delete => [
                  '~/Library/Preferences/com.malwarebytes.antimalware.plist',
                  '~/Library/Caches/com.malwarebytes.antimalware',
                  '~/Library/Application Support/com.malwarebytes.antimalware',
                  '~/Library/Saved Application State/com.malwarebytes.antimalware.savedState'
                 ]
```

File: Casks/malwarebytes-anti-malware.rb [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+  url "https://data-cdn.mbamupdates.com/v1/mbam-mac/data/mbam-mac-#{version}.zip"
   name 'Malwarebytes Anti-Malware for Mac'
-  appcast 'http://data-cdn.mbamupdates.com/v1/mbam-mac/updates.xml',
+  name 'AdwareMedic' # former name
+  appcast 'https://data-cdn.mbamupdates.com/v1/mbam-mac/updates.xml',
           :sha256 => '065d31d146031074b4e249a8994fffedf589fffbc114ad3c6bb98c2195878de4',
           :format => :sparkle
   homepage 'https://www.malwarebytes.org/antimalware/mac/'
   license :gratis
+  depends_on :macos => '>= :lion'
```
Comment (vitorgalvao) [inline Casks/malwarebytes-anti-malware.rb]: Should have a blank line on top, and be the last stanza.

File: Casks/malwarebytes-anti-malware.rb [hunk 3]
```diff
@@ -1,14 +1,22 @@
 cask :v1 => 'malwarebytes-anti-malware' do
   version '1.0.2.8'
-  sha256 '90f2109cf4f1150c9888ec0ed1110407880b1fc39e17f8daeba862878bd9a796'
+  sha256 '065d31d146031074b4e249a8994fffedf589fffbc114ad3c6bb98c2195878de4'
 
-  url "https://mbam-mac-dl.malwarebytes.org/MBAM-Mac-#{version}.dmg"
+  url "https://data-cdn.mbamupdates.com/v1/mbam-mac/data/mbam-mac-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/malwarebytes-anti-malware.rb]: Needs a `  # mbamupdates.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by Eitot** [comment]
Updated and squashed

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**